tests/src/ut_test_kvp.c

```diff
@@ -378,4 +412,18 @@ void register_kvp_functions( void )
 
     UT_add_test(gpKVPSuite4, "kvp read negative", test_ut_kvp_get_field_without_open);
 
-}
\ No newline at end of file
+    gpKVPSuite5 = UT_add_suite("ut-kvp - test get profile function", NULL, NULL);
+    assert(gpKVPSuite5 != NULL);
+
+    UT_add_test(gpKVPSuite5, "kvp profile instance create", test_ut_kvp_getPlatformProfile);
+
+    gpKVPSuite6 = UT_add_suite("ut-kvp - test main functions YAML Decoder from file passed as argument", test_ut_kvp_getPlatformProfileInstance, NULL);
```

Comment:
do you need this? Does Suite6 use the input file specified by `-p` it shouldn't? I thought it was loading it itself internally.